src/plugins/sails.js

```diff
@@ -190,3 +191,15 @@ io.socket.on('leftGame', function (evData) {
     store.commit('otherLeftGame', evData.id);
   }
 });
+
+//////////////////
+// Connectivity //
+//////////////////
+io.socket.on('connect', () => {
+  // Request latest game state if socket reconnects during game
+  const { username } = store.state.auth;
+  const inGame = router.currentRoute.value.name === ROUTE_NAME_GAME;
+  if (inGame) {
```

Comment:
I’d flip this to return early, otherwise do this logic after the if check.

Reply:
Are you suggesting this?
```
  if (!inGame) {
    return;
  }
  return store.dispatch('requestReauthenticate', { username });
```

Reply:
Yeah exactly, I always order a fast fail approach.

Reply:
For anyone else reading, my personal reasoning is that I like to see the return cases clearly defined, tends to make more sense to me when working through the happy path as the final return. There’s probably a case to be made for it being more performant in some cases too, depends on the code.

Reply:
in the fullness of time, I think we'll actually wind up beefing this up to do different logic depending on the page i.e. get the gameList from the `HomeView` vs rehydrating the game data when you're in a game. So if and when we get there we may wind up shifting to a switch and/or something that's more case driven vs a sortof on/off logic like we have now